app/models/user.rb

```diff
@@ -134,7 +144,7 @@ def update_with_phone_number_authentication(params)
     )
 
     transaction do
-      update!(user_params) && phone_number_authentication.update!(phone_number_authentication_params)
+      update(user_params) && phone_number_authentication.update!(phone_number_authentication_params)
```

Comment:
Why this change?

Reply:
This page returned a 400 earlier if an update failed, we want to show the errors on the form now. 
![image](https://user-images.githubusercontent.com/16774200/91159103-389e0200-e6e5-11ea-8d35-ad7f6cb49d6d.png)